Test Case Failed - TestRelativeChange::test_year_iterable
Relevant prompt Section - years (iterable): An iterable containing the years corresponding to the last axis of `dataset`.
Reason - This test case failed because the generated solution attempts to use the index() function on years. According to the prompt, years is defined as an iterable, which can include various data types such as lists, arrays, and sets etc. However, the index() method is not supported by all iterable types.The usuage of index() leads to  incorrect functional implementation ultimately leading to failure of test case. 